py-polars/src/lib.rs

```diff
@@ -376,14 +372,63 @@ fn polars(py: Python, m: &Bound<PyModule>) -> PyResult<()> {
     // Build info
     m.add("__version__", env!("CARGO_PKG_VERSION"))?;
     #[cfg(feature = "build_info")]
-    m.add(
-        "__build__",
-        pyo3_built!(py, build, "build", "time", "deps", "features", "host", "target", "git"),
-    )?;
+    add_build_info(py, m)?;
 
     // Plugins
     m.add_wrapped(wrap_pyfunction!(functions::register_plugin_function))
         .unwrap();
 
     Ok(())
 }
+
+#[cfg(feature = "build_info")]
+fn add_build_info(py: Python, m: &Bound<PyModule>) -> PyResult<()> {
+    use pyo3::types::{PyDict, PyString};
+    let info = PyDict::new_bound(py);
+
+    let build = PyDict::new_bound(py);
```

Comment:
Let's call this variable `compiler` too, then?

```suggestion
    let compiler = PyDict::new_bound(py);
```